wifi: Add and tweak confguration for 54L memory baseline (#19981)
nrfconnect/sdk-nrf: samples/wifi/sta/README.rst

See :ref:`app_power_opt` for more information on power management testing and usage of the PPK2.

Performance testing
*******************

The sample can be used to test the performance of the Wi-Fi connection.
The performance tuning is done to achieve a trade-off between memory usage and performance.

> The overlay file is located at :file:`overlay-zperf.conf` to run the performance test.

Reviewer: ```suggestion
You can use the :file:`overlay-zperf.conf` file to run the performance test.
```

Default build without the overlay is used to memory footprint testing and analysis.

> The overlay should be enabled to run the performance test corresponding to the memory footprint.

Reviewer: ```suggestion
The overlay must be enabled to run the performance test corresponding to the memory footprint.
```
